test/integration/scheduler/taint_test.go

```diff
@@ -125,6 +125,8 @@ func TestTaintNodeByCondition(t *testing.T) {
 	// Waiting for all controller sync.
 	internalInformers.Start(controllerCh)
 	internalInformers.WaitForCacheSync(controllerCh)
+	informers.Start(controllerCh)
```

Comment:
I thin the `informerFactory` was started in `initTestSchedulerWithOptions` :)

Reply:
The extension.v1veta1.DaemonSet is not started by infomers (also `informerFactory` in `initTestSchedulerWithOptions`); it's better to handle it :)

Reply:
Yeap, noticed that your put an explicit daemonset informer start in initTestScheduler().

My fix although achieves the same effect :)

Despite of your or my fix, there are still flaky failures - quite random. I'm suspecting it's related with limit controlling of node lifecycle manager as it enters a fully disrupted mode.

Reply:
xref https://github.com/kubernetes/kubernetes/pull/67864#issuecomment-417579147